src/tests/order/test_increase_order_utils.cairo

```diff
@@ -0,0 +1,60 @@
+use starknet::ContractAddress;
+use snforge_std::{start_mock_call, stop_mock_call};
+
+use satoru::data::data_store::IDataStoreDispatcherTrait;
+use satoru::nonce::nonce_utils::{get_current_nonce, increment_nonce, compute_key};
+use satoru::tests_lib::{setup, teardown};
+use satoru::oracle::oracle::{IOracleSafeDispatcher, IOracleDispatcher, IOracleDispatcherTrait};
+use satoru::order::{
+    error::OrderError, order::{Order, SecondaryOrderType, OrderType, DecreasePositionSwapType},
+};
+use satoru::order::increase_order_utils::{validate_oracle_block_numbers};
+
+// TODO - Add tests for process_order
```

Comment:
Tests only cover `validate_oracle_block_numbers`, added todo for `process_order` to implement it once `increase_position_utils::increase_position` is implemented